packages/plugin-core/src/commands/NoteLookupCommand.ts

```diff
@@ -422,6 +422,10 @@ export class NoteLookupCommand
                 item.traits = [journalTrait];
               }
             }
+          } else if (
```

Comment:
good point, let me make an adjustment for this.

Reply:
the problem with putting the logic in plugin-core is that it won't apply to notes created via goto note or the CLI. for settings that affect note state, can we add the logic in the engine? eg. engine.writeNote

Reply:
It's not easy to do that right now - there isn't a way to pass in info to `DendronEngineClient` regarding journal/scratch/task flags, note-traits, or anything like that right now.  We'll have to do some re-working of how these modifiers work in order to properly support them in both CLI and plugin.

I'm inclined to just support in plugin right now, or else add some replicated logic in CLI side, which is on parity with how journal notes also work right now.

Reply:
@SeriousBug  - actually on second thought, I think it makes more sense to not apply this to journal notes.  Right now, journal notes will only use a date portion for the title, so I don't think it makes sense to alter that behavior.  I.e. a journal note with fname 'foo.bar.2020.01.01' produces a note with title '2020-01-01', so it doesn't make sense to alter this title to anything like 'Foo Bar 2020-01-01' since the non-numerical portion was already getting dropped.  I'll update the docs.